### PR TITLE
Fix Broken Header Link

### PR DESCRIPTION
**Pull request by julieg18**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -271,7 +271,7 @@ <h2 class="additional-info__title">We know a lot</h2>
         </ul>
       </section>
 
-      <section class="donations">
+      <section class="donations" id="donations">
```
Comment (tantany) [inline index.html:274]: I think that in BEM we don't use id's.

File: index.html [hunk 2]
```diff
@@ -43,7 +43,7 @@
           >
         </li>
         <li>
-          <a href="#how-to-help" class="header__link header__link_border"
+          <a href="#donations" class="header__link header__link_border"
```
Comment (tantany) [inline index.html:46]: Do we use id in BEM?

File: index.html (hunk 1 above)
Comment (julieg18) [reply index.html:274]: We don't use IDs for styling, but I don't know of another way to make a local link.

File: index.html (hunk 2 above)
Comment (julieg18) [reply index.html:46]: That's not an id, that's a marker for a local link.

Comment (tantany) [reply index.html:46]: You can use href="#". I don't think you need the  id="donations" and the name after the "#"

Comment (julieg18) [reply index.html:46]: Ok, though that would leave the link not working. Wouldn't it be more logical if the "Help Us" link took us to the donations part of the site?

Comment (tantany) [reply index.html:46]: On second thought, you are right. Leave it as it is.

Comment (julieg18) [reply index.html:46]: 👍

**Pull request merged**